pages/workspaces/new.tsx

```diff
@@ -30,7 +30,7 @@ const NewWorkspace = () => {
       // TODO: send members list
       members: [],
       sessionToken: sessionToken!,
-      repos: trackedRepos.map((repo) => ({ full_name: repo })),
+      repos: Array.from(trackedRepos).map((repo) => ({ full_name: repo })),
```

Comment:
`Array.from` can take a mapping function as its second argument.

Reply:
Good call. I forgot about that. Thanks @NsdHSO!